dev/infra_server_objects_count_collector.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script gets the count of the chef server objects.
+
+OPTIONS:
+   -h      Shows the help message
+   -S      Chef Server url (Eg: https://chef-server.example.com)
+   -K      Path to the pivotal user key
+   -F      Output file path
+EOF
+}
+
+check_binary_install_jq() {
+  if ! which "$1" > /dev/null; then
+    # Using a subshell to redirect output to stderr. It's cleaner this way and will play nice with other redirects.
+    echo "jq is required to execute the script. Intalling it on this machine now.."
+    if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        sudo apt-get install jq
```

Comment:
This will work for `Debian` based distros but will fail in RHEL.

Reply:
Though I am using Debian, it didn't work for me. No valid reason provided.

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/44021874/227970966-32f92b3e-b530-47d2-87db-3ecccb0632fa.png">

Reply:
As discussed with kallol, as this is a tricky part we are just showing a warning to the user now to install jq.